icu4c/source/data/cldr-icu-readme.txt

```diff
@@ -236,6 +236,7 @@ make check 2>&1 | tee /tmp/icu4c-newData-makeCheck.txt
 # Note that if the new data has any differences in structure, you will have to
 # update test/testdata/structLocale.txt or /tsutil/cldrtest/TestLocaleStructure
```

Comment:
Instructions don't say how to update structLocale.txt - as per FIXME added below, is it manually maintained now?